tsfc/fiatinterface.py

```diff
@@ -58,7 +58,8 @@
     "Nedelec 2nd kind H(curl)": FIAT.NedelecSecondKind,
     "TensorProductElement": FIAT.TensorProductElement,
     "Raviart-Thomas": FIAT.RaviartThomas,
-    "TraceElement": FIAT.HDivTrace,
+    "TraceElement": FIAT.TraceHDiv,
```

Comment:
I assume you have this for compatibility with old code. It would probably be better to register that alias in UFL.